API/projectClass/resolutionsClass.m

```diff
@@ -1,18 +1,31 @@
 classdef resolutionsClass < handle
-    % Resolutions are defined in a two stage process. Firstly we define the
-    % actual fitted parameters. These are held in a 'ParametersClass'
-    % table. Then, we group these into the resolutions themselves using a
-    % multiTypeTable. So, we can then use the resolution parameters to
-    % either define resolution as  constant, data or a function. 
+    % Resolutions are defined in a two-stage process. Firstly, we define the
+    % actual fitted parameters. These resolution parameters are held in a ``parametersClass`` object 
+    % and can be used define resolutions. 
     % 
-    % For constant only one parameter is supplied to multi type table. 
+    % For constant resolution, only one resolution parameter is supplied. 
```

Comment:
This should be either "For a constant resolution, . . ." or "For constant resolutions, . . ."